src/legacy/core_plugins/table_vis/public/agg_table/agg_table.js

```diff
@@ -128,7 +130,7 @@ uiModules
             const isDate = _.get(dimension, 'format.id') === 'date' || _.get(dimension, 'format.params.id') === 'date';
             const isNumeric = _.get(dimension, 'format.id') === 'number' || _.get(dimension, 'format.params.id') === 'number';
```

Comment:
can you use here your function `isNumeric`?

Reply:
I don't think I can... Here, I don't have the full aggConfig available to me to key off of but in the other location, the array order of the columns and config objects differs so I can't use this method.